scripts/setup.sh

```diff
@@ -1,14 +1,50 @@
 #!/usr/bin/env bash
-
 set -ex
 
-apt-get update
-source /opt/ros/$ROS_DISTRO/setup.sh
+if [[ -z "$ROS_DISTRO"  ]]
+then
+		#timezone is required for ROS Installation.
+		if [ ! -f "/etc/timezone" ];
+		then
+			export TZ="US/Pacific"
+			ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+		fi
+		#Install ROS Prerequisites
+		apt update
+		apt-get install -y lsb-release gnupg2 curl && apt-get clean all
+
+        #Install ROS Melodic
+        echo "installing ROS Melodic"
+        sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
+        apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
+        curl -sSL 'http://keyserver.ubuntu.com/pks/lookup?op=get&search=0xC1CF6E31E6BADE8868B172B4F42ED6FBAB17C654' | apt-key add -
+        apt update
+        apt install -y ros-melodic-desktop-full
+        source /opt/ros/melodic/setup.bash
+
+elif [ $ROS_DISTRO == "melodic" ]
+then
+        echo "ROS Melodic is already installed. Skipping ROS Installation"
+else    #Warning your ROS Distro $ROS_DISTRO is not supported for the Hello World Sample Application
+        #it may not work.
+        echo "Warning your ROS Distro $ROS_DISTRO is not supported for AWS RoboMaker Hello World Sample Application ros1 branch."
```

Comment:
* Maybe we should give an option to install melodic anyway or give some direction how they can proceed from here. 
* We should exit from the script and not proceed to the next steps.

Reply:
How do we feel about installing a second distro of ROS?

Reply:
What do you mean by second distro?

Reply:
We shouldn't install another ROS distro on top of the customers installation (I don't think it would even work).. we should exit the installation with a descriptive error that suggests possible mitigations.

Reply:
We shouldn't install another ROS distro on top of the customers installation (I don't think it would even work).. we should exit the installation with a descriptive error that suggests possible mitigations before proceeding.